packages/nartus_ui_package/lib/widgets/bottom_sheet/nartus_bottom_sheet.dart

```diff
@@ -0,0 +1,109 @@
+      {required this.title,
+      required this.content,
+      required this.primaryButtonText,
+      required this.onPrimaryButtonSelected,
+      Key? key,
+      this.iconPath,
+      this.secondaryButtonText,
+      this.onSecondButtonSelected})
+      : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      padding: const EdgeInsets.fromLTRB(32, 40, 32, 58),
+      child: Column(
+        mainAxisSize: MainAxisSize.min,
+        mainAxisAlignment: MainAxisAlignment.center,
+        children: <Widget>[
+          if (iconPath != null)
+            ExcludeSemantics(
+                child: Padding(
+              padding: const EdgeInsets.only(bottom: NartusDimens.padding24),
+              child: SvgPicture.asset(
+                iconPath!,
+                fit: BoxFit.scaleDown,
+                width: NartusDimens.size80,
+                height: NartusDimens.size80,
+              ),
+            )),
+          Padding(
+              padding: const EdgeInsets.only(bottom: NartusDimens.padding8),
+              child: Text(
+                title,
+                style: Theme.of(context)
+                    .textTheme
+                    .headlineLarge
+                    ?.copyWith(color: NartusColor.dark),
+              )),
+          Padding(
+              padding: const EdgeInsets.only(bottom: NartusDimens.padding24),
+              child: Text(
+                title,
+                style: Theme.of(context)
+                    .textTheme
+                    .bodyMedium
+                    ?.copyWith(color: NartusColor.dark),
+              )),
+          Padding(
+            padding: const EdgeInsets.only(bottom: NartusDimens.padding30),
+            child: NartusButton.primary(
+              label: primaryButtonText,
+              onPressed: onPrimaryButtonSelected,
+            ),
+          ),
+          if (secondaryButtonText != null)
+            NartusButton.secondary(
+              label: secondaryButtonText,
+              onPressed: onSecondButtonSelected,
+            )
+        ],
+      ),
+    );
+  }
+}
+
+extension IdBottomSheet on BuildContext {
+  void showIDBottomSheet(
+      {required String title,
+      required String content,
+      required String primaryBtnText,
+      required VoidCallback onPrimaryBtnSelected,
+      String? iconPath,
+      String? secondaryBtnText,
+      VoidCallback? onSecondaryBtnSelected,
+      bool isDismissible = true}) {
+    showModalBottomSheet(
+        context: this,
+        isDismissible: isDismissible,
+        shape: const RoundedRectangleBorder(
+            borderRadius: BorderRadius.vertical(top: Radius.circular(20))),
+        builder: (BuildContext builder) {
+          return NartusBottomSheet(
+            iconPath: iconPath,
+            title: title,
+            content: content,
+            primaryButtonText: primaryBtnText,
+            onPrimaryButtonSelected: onPrimaryBtnSelected,
```

Comment:
Please use the name onPrimaryButtonPressed, onSecondaryButtonPress, and onTextButtonPressed